jest-common/pom.xml

```diff
@@ -69,6 +69,11 @@
         </dependency>
 
         <dependency>
+            <groupId>org.easytesting</groupId>
```

Comment:
The assertions used (contains/null/equals) really falls short of justifying the need for this additional library as we already have junit. Let's stick with junit for now.